lib/tasks/lint.rb

```diff
@@ -7,18 +7,31 @@
   require_relative 'rake_helper'
   include RakeHelper
 
+  DEFAULT_PATTERNS = [
+    "apps/**/*.rb",
+    "lib/**/*.rb",
+    "nginx_stage/**/*.rb",
+    "ood-portal-generator/**/*.rb",
+    "spec/**/*.rb",
+  ]
+
   begin
     require 'rubocop/rake_task'
     RuboCop::RakeTask.new(:rubocop, [:path]) do |t, args|
```

Comment:
I'm not sure where this object comes from. Do we use `rubocop-rake`? I don't see it in any `Gemfile` and I can't find any docs around this method of creating the object outside of what I found in the `rubocop-rake` repo's examples.

Reply:
https://www.rubydoc.info/gems/rubocop/RuboCop/RakeTask - this is a pretty common pattern for tools that provide custom Rake tasks to allow customizations of those Rake tasks.  We actually use this pattern in the OnDemand Packaging repo.

Reply:
I couldn't find this referenced on rubocop docs aside from references to how to enable rubocop plugins via Rake tasks.  Such as this: https://docs.rubocop.org/rubocop-rails/usage.html#rake-task